README.md

```diff
@@ -6,15 +6,14 @@
 [![License](https://img.shields.io/hexpm/l/prima_auth0_ex.svg)](https://github.com/primait/auth0_ex/blob/master/LICENSE.md)
 [![Last Updated](https://img.shields.io/github/last-commit/primait/auth0_ex.svg)](https://github.com/primait/auth0_ex/commits/master)
 
-An easy to use library to authenticate machine-to-machine communications through Auth0.
+An easy-to-use library to authenticate machine-to-machine communications through Auth0.
 
 Supports both retrieval of JWTs and their verification and validation.
 
 ## Table of contents
 
 - [Installation](#installation)
 - [Configuration](#configuration)
-  - [I am a DevOps, what do I have to do?](#i-am-a-devops-what-do-i-have-to-do)
```

Comment:
Link here? Same as before?

Reply:
good catch @cottinisimone! @cpiemontese can you fix pliz?